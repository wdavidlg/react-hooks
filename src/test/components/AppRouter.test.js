const { mount } = require("enzyme")
const { default: AppRouter } = require("../../components/09-useContext/AppRouter")
const { default: UserContext } = require("../../components/09-useContext/UserContext")


describe('Pruebas en <AppRouter />', () => {
    
    const user = {
        id: 1,
        name: 'will',
        emai: 'will@will.com'
    }

    const wrapper = mount(
        <UserContext.Provider value={{
            user
        }}>
            <AppRouter/>
        </UserContext.Provider>
    )

    test('debe de mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
        
    })
    
})
