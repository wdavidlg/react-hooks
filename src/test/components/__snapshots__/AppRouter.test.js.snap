// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AppRouter /> debe de mostrarse correctamente 1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <div
      className="container-fluid"
    >
      <a
        className="navbar-brand"
        href="/"
        onClick={[Function]}
      >
        Context
      </a>
      <div
        className="collapse navbar-collapse"
        id="navbarNavAltMarkup"
      >
        <div
          className="navbar-nav"
        >
          <a
            className="nav-link active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
          <a
            className="nav-link active"
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
          <a
            className="nav-link active"
            href="/login"
            onClick={[Function]}
          >
            Login
          </a>
        </div>
      </div>
    </div>
  </nav>
  <div
    className="container mt-5"
  >
    <div>
      <h2>
        HomeScreen
      </h2>
      <hr />
      <pre>
        {
    "id": 1,
    "name": "will",
    "emai": "will@will.com"
}
      </pre>
    </div>
  </div>
</div>
`;
